saskatoon/sitebase/templates/app/forms/property_create_public.html

```diff
@@ -15,6 +15,7 @@
                     <div class="form-example-wrap">
                         <div class="cmp-tb-hd">
                             <h2>Add a new property</h2>
+                            <small class="text-muted">This form is for persons with a tree on their property and community organizations should contact us directly</small>
```

Comment:
I'm not sure if this notice is really necessary here. It's okay if we receive pending properties from organizations as long as we have some kind of way of knowing (e.g. core members give specific instructions on how to add a note in the name or additional info box). Btw we could already edit the `help_text` in the `additional_info` box to hint at it.
This way core members could tick the `Contact Person` instead of `Property owner` when they use the register owner form. 
(They will then need to manually create an Organization, link it to the contact person, and finally manually add the organization as Owner). We will make sure to clarify this flow in our guides with core members.

Also I'm wondering if we could improve the title for something like <h2>Register a property for the {current_year} season</h2>
Some properties aren't "new" since they can check the "have you registered" before box.
@tristanlatr what do you think? We could also open a separate issue to finalize the look/labels/translations in the public form

Reply:
@2ynn got it, thanks for elaborating the process of how organization creation works.

I'm removing the last commit for now that deals with this help text. Other than this, I think my other changes are ready to be reviewed.

We can work on the look/labels/translations related changes on a different issue as you have suggested.